pulse/broker/src/main/java/gov/ca/emsa/pulse/broker/entity/PatientEntity.java

```diff
@@ -28,11 +30,13 @@
 	@Column( name = "id", nullable = false )
 	private Long id;
 	
-	@Column(name="given_name")
-	private String givenName;
+	@Column(name = "patient_name_id")
+	private Long patientNameId;
 	
-	@Column(name = "family_name")
-	private String familyName;
+	@OneToOne(optional = true, fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+	@Fetch(FetchMode.JOIN)
+	@JoinColumn(name = "patient_name_id", unique=true, nullable = true, insertable=false, updatable= false)
+	private PatientNameEntity patientName;
```

Comment:
The PatientEntity can have more than one PatientNameEntity; the `patient_name_id` column shouldn't be in PatientEntity. PatientNameEntity should have a foreign key relationship to PatientEntity